styles.css

```diff
@@ -0,0 +1,420 @@
+body {
+  background: #f4f5f6;
+  font-family: "Roboto Condensed", sans-serif;
+  margin: 0;
+}
+
+.container_wrapper {
+  max-width: 1300px;
+  margin: auto;
+  background: #fff;
+}
+
+a {
+  text-decoration: none;
+  color: #161616;
+}
+
+.article-post a {
+  text-decoration: none;
+}
+
+.social li a {
+  font-size: 1.3rem;
+  padding-right: 1rem;
+  color: #666;
+}
+
+.footer-navigation li a {
+  padding: 0 0.8rem;
+  border-right: 1px solid #e5e5e5;
+  color: #666;
+  font-family: sans-serif;
+  font-size: 0.9rem;
+}
+
+.container_wrapper .nav-item a {
+  font-weight: 700;
+  color: #000;
+  font-size: 1.25rem;
+  margin-bottom: 0;
+  padding: 0 1rem;
+  border-right: 1px solid #e5e5e5;
+}
+
+.logo-area {
+  background: #f72210;
+  margin: 0;
+}
+
+.current-date {
+  font-weight: 100;
+  color: #fff;
+}
+
+.signin-btn {
+  background: transparent;
+  color: #fff;
+  font-weight: bold;
+}
+
+.subscribe-btn {
+  background: #fff;
+  border-radius: 0;
+  padding: 0.3rem 1.2rem;
+  font-weight: bold;
+  color: #f72210;
+}
+
+.search-bar {
+  border: none;
+  background: transparent;
+  font-size: 1rem;
+}
+
+.navbar {
+  border-bottom: 1px solid #e5e5e5;
+  margin: 0;
+}
+
+nav ul li {
+  list-style: none;
+}
+
+main {
+  padding: 2rem;
+}
+
+.featured-col img {
+  width: 100%;
+}
+
+.featured {
+  max-width: 98%;
+  margin: auto;
+}
+
+.fa-search {
+  position: absolute;
```

Comment:
Consider using a `Bootstrap` class name for this selector element and in all other occurrences instead.

Bootstrap class names can be used instead. See [Position](https://getbootstrap.com/docs/4.1/utilities/position/) in bootstrap documentation.

> Remember to use Bootstrap CSS classes wherever it is possible - this is the main point of this project.

#### Please, lets do this for all other occurrences. :blush: